packages/contentful-migrations/core/0000000000003-create-t-page-migration.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-env node */
+const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+module.exports = withHelpers(async (migration, _context) => {
+  const tPage = migration
+    .createContentType('t-page')
+    .name('Template: Page')
+    .description('Template for a regular web page')
+    .displayField('internal_name');
+
+  tPage
+    .createField('internal_name')
+    .name('Internal name')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('title')
+    .name('Page title')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('slug')
+    .name('Slug')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('parent_page')
+    .name('Parent page')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['folder', 't-page'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  tPage
+    .createField('seo_title')
+    .name('SEO title')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('seo_description')
+    .name('SEO description')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('share_image')
+    .name('Share image')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkMimetypeGroup: ['image'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Asset');
+
+  tPage
+    .createField('no_index')
+    .name('Hide page from search engines? (noindex)')
+    .type('Boolean')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('no_follow')
+    .name('Exclude links from search rankings? (nofollow)')
+    .type('Boolean')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('theme')
+    .name('Theme')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        in: ['light', 'dark'],
+      },
+    ])
+    .disabled(true)
+    .omitted(true);
+
+  tPage
+    .createField('stage')
+    .name('Stage')
+    .type('Link')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        linkContentType: ['m-stage'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  tPage
+    .createField('modules')
+    .name('Modules')
```

Comment:
Just checked contentful after running all migrations and i think that `Content` would be easier to understand for content editors  than `Modules`

Reply:
Done